flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/DynamicFilteringITCase.java

```diff
@@ -118,8 +124,10 @@ public void before() throws Exception {
                         dataId3));
     }
 
-    @Test
-    public void testSimpleDynamicFiltering() {
+    @ParameterizedTest(name = "mode = {0}")
+    @MethodSource("parameters")
```

Comment:
change them to class level ?

Reply:
Class level parameterized tests are now not supported in junit 5.